src/components/SignatureRequest.js

```diff
@@ -36,7 +36,10 @@ const SignerRequest = React.memo(function SignerRequest({
 
   const { statusText, statusTextColor } = useMemo(() => {
     if (signingError) {
-      return { statusText: 'Signature failed', statusTextColor: theme.negative }
+      return {
```

Comment:
This is what prettier did for me, but maybe I have an older version?

Reply:
That's the correct format